orca-web/orca-web.gradle

```diff
@@ -33,6 +33,8 @@ dependencies {
   implementation("org.springframework.boot:spring-boot-starter-actuator")
   implementation("net.logstash.logback:logstash-logback-encoder")
 
+  implementation(project(":keiko-spring"))
```

Comment:
Without this Orca wasn't able to load up `ObjectMapperSubtypeProperties` from `keiko-spring` on startup.

Reply:
whoop. thanks